examples/ini/grafana.ini

```diff
@@ -0,0 +1,679 @@
+##################### Grafana Configuration Defaults #####################
```

Comment:
The example is a bit long. Readability would already improve by removing the empty variables.